wallet/service.go

```diff
@@ -64,6 +69,29 @@ func (service *Service) GetOrCreateWallet(ctx context.Context, walletID uuid.UUI
 	return wallet, nil
 }
 
+// GetAndCreateMemberWallets all wallets associated with the wallet id provided are accounted for
+func (service *Service) GetAndCreateMemberWallets(ctx context.Context, walletID uuid.UUID) (*wallet.Info, error) {
+	// may be unnecessary to fetch
+	wallets, err := service.LedgerClient.GetMemberWallets(ctx, walletID)
+	if err != nil {
+		return nil, err
+	}
+	if len(*wallets) == 0 {
```

Comment:
How does this work? Is this checking the length of the address?

Reply:
This actually is dead code path, removing in next commit.

Reply:
https://github.com/brave-intl/bat-go/pull/314/commits/d9ad61f5ecbbf6e774a35d6481720162bacc3f87